WHITEPAPER.md

```diff
@@ -130,14 +133,35 @@ it is good to check the basics of your system:
 
 * Security: Look at the [OWASP](https://www.owasp.org/index.php/Main_Page))
   project which provides good security approaches for various use-cases
-* ...
-
-
-
-
-#### Do I need to get started in production
-
-While we may want this, starting in prod may not fit "getting started scenarios".
+* Logging: Your system needs to leave traces about what it is doing so you
+  can investigate it. Central logging is often a prime idea to make this
+  process much smoother and faster.
+* Monitoring: Your system constantly sends pulses about its current state. Those
+  signals tell you something about how it fares when you capture the right
+  metrics.
+* Automation: Whenever you deal with enough complexity, automation can save your
+  day. It is a wide topic and there are many levels of automation but starting
+  with Continuous Integration is a great starting point. Before moving on to
+  Continuous Delivery, Deployment and perhaps even GitOps.
+* Testing: You want to move fast but with a high degree of confidence. Testing
+  is a must-do practice to achieve it.
+
```

Comment:
Maybe we can add alerting too ?

Reply:
Indeed!

Reply:
I've seen the word GitOps passed around quite a bit but I fail to see how it's different from regular CI/CD; other than that it's strictly ops focused. If I'm being too nit-picky let me know it just seems like terms that are trying to find establishment probably don't have a place in a whitepaper.